be/src/olap/field.h

```diff
@@ -298,6 +298,11 @@ class Field {
     Field* get_sub_field(int i) const { return _sub_fields[i].get(); }
     size_t get_sub_field_count() const { return _sub_fields.size(); }
 
+    void set_precision(int32_t precision) { _precision = precision; }
+    void set_frac(int32_t frac) { _frac = frac; }
+    int32_t precision() const { return _precision; }
+    int32_t frac() const { return _frac; }
```

Comment:
Does frac the same as scale? If it is true, why not use scale?

Reply:
yes, frac is the same as scale. All code uses frac in src/olap, so I used frac here.